src/Microsoft.Framework.Runtime.Abstractions/ILibraryManager.cs

```diff
@@ -11,24 +11,24 @@ namespace Microsoft.Framework.Runtime
     /// </summary>
     public interface ILibraryManager
```

Comment:
Talked to @davidfowl . We'll do that and get it in Monday to break the world early.